lib/matplotlib/axes/_base.py

```diff
@@ -4388,18 +4387,13 @@ def get_tightbbox(self, renderer, call_axes_locator=True,
                 bb.append(bb_yaxis)
 
         self._update_title_position(renderer)
+
         axbbox = self.get_window_extent(renderer)
         bb.append(axbbox)
```

Comment:
this already adds self.get_window_extent(), so we don't need to add it again below.